modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderImport.testcase

```diff
@@ -197,6 +197,138 @@ definition {
+				paymentMethod = "money-order",
+				paymentStatus = 2);
+		}
+
+		task ("And the buyer is importing the order from Orders") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+
+			CommerceAcceleratorsInitializer.createNewOrderFromPendingOrders();
+
+			CommerceOrders.importOrderFromType(orderType = "Orders");
+
+			SelectFrameTop();
+
+			IFrame.selectModalFrame();
+
+			Button.click(button = "Select");
+		}
+
+		task ("And can see data set is filled with the following data in the preview modal and only four items are displayed") {
+			Pagination.changePagination(itemsPerPage = "4 items");
+
+			CommerceEntry.viewPaginationResults(results = "Showing 1 to 4 of 13");
+
+			CommerceOrders.assertOrderImportPreviewEntry(
+				importStatus = "OK",
+				productName = "ABS Sensor",
+				productQuantity = 1,
+				productSku = "MIN93015",
+				totalPrice = "50.00",
+				unitPrice = "50.00");
+
+			CommerceOrders.assertOrderImportPreviewEntry(
+				importStatus = "OK",
+				productName = "Premium Brake Fluid",
+				productQuantity = 1,
+				productSku = "MIN93017",
+				totalPrice = "100.00",
+				unitPrice = "100.00");
+
+			CommerceOrders.assertOrderImportPreviewEntry(
+				importStatus = "OK",
+				productName = "Wear Sensors",
+				productQuantity = 1,
+				productSku = "MIN93027",
+				totalPrice = "60.00",
+				unitPrice = "60.00");
+		}
+
+		task ("When the user go to next page") {
+			Click(locator1 = "ACUtils#NEXT_PAGES");
+		}
+
+		task ("Then four different items are displayed") {
+			CommerceEntry.viewPaginationResults(results = "Showing 5 to 8 of 13");
+
+			CommerceOrders.assertOrderImportPreviewEntry(
+				importStatus = "OK",
+				productName = "Calipers",
+				productQuantity = 1,
+				productSku = "MIN93021",
+				totalPrice = "90.00",
+				unitPrice = "90.00");
+
+			CommerceOrders.assertOrderImportPreviewEntry(
+				importStatus = "OK",
+				productName = "Premium Brake Pads",
+				productQuantity = 1,
+				productSku = "MIN93019",
+				totalPrice = "29.00",
+				unitPrice = "29.00");
+
+			CommerceOrders.assertOrderImportPreviewEntry(
+				importStatus = "OK",
+				productName = "Brake Rotors",
+				productQuantity = 1,
+				productSku = "MIN93020",
+				totalPrice = "40.00",
+				unitPrice = "40.00");
+
+			AssertElementNotPresent(
+				key_columnNumber = 1,
+				key_productName = "Premium Brake Fluid",
+				key_value = "MIN93017",
+				locator1 = "CommerceEntry#TABLE_SEARCH_BY_NAME_AND_COLUMN_CONTENT");
+
+			AssertElementNotPresent(
+				key_columnNumber = 1,
+				key_productName = "ABS Sensor",
+				key_value = "MIN93015",
+				locator1 = "CommerceEntry#TABLE_SEARCH_BY_NAME_AND_COLUMN_CONTENT");
+
+			AssertElementNotPresent(
+				key_columnNumber = 1,
+				key_productName = "Wear Sensors",
+				key_value = "MIN93027",
+				locator1 = "CommerceEntry#TABLE_SEARCH_BY_NAME_AND_COLUMN_CONTENT");
+
+			IFrame.closeFrame();
+		}
```

Comment:
The assertion of the previous page arrow is missing. Also paginating by directly clicking on a page number is missing